Makefile

```diff
@@ -29,3 +29,7 @@ stretch-slim: build_10-stretch-slim build_12-stretch-slim build_14-stretch-slim
 14-stretch-slim:
 	docker build -t local/articulate-node:14-stretch-slim 14-stretch-slim
 .PHONY: 14-stretch-slim
+
+12-buster-slim:
```

Comment:
Can we create a buster-slim target and add this to that and then add that to the all target?

Reply:
Looks like this needs to be cleaned up a little bit. I can take care of that outside this PR